### PR TITLE
Fix editor

### PR DESCRIPTION
**Pull request by annakocheshkova**
Remove placeholders from secrets;
Turned off Startup type in AppCenterBehaviorAdvanced.
Now, if AppCenterBehaviorAdvanced is attached, the startup ttype changes to *Both*. Otherwise it's *AppCenter*.

### CODE REVIEW

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs [hunk 1]
```diff
@@ -111,8 +119,15 @@ static void AddStartupCodeToiOS()
...
-        settingsMaker.SetTransmissionTargetToken(advancedSettings.TransmissionTargetToken);
+        if (advancedSettings != null)
+        {
+            settingsMaker.SetStartupType((int)advancedSettings.AppCenterStartupType);
+            settingsMaker.SetTransmissionTargetToken(advancedSettings.TransmissionTargetToken);
+        }
+        else
+        {
+            settingsMaker.SetStartupType((int)StartupType.AppCenter);
+        }
```
Comment (max-mironov) [inline Assets/AppCenter/Editor/AppCenterPreBuild.cs:130]: lines 75-83 and 122-130 are the same? Can we extract to method?

File: Assets/AppCenter/Editor/AppCenterSettingsEditorAdvanced.cs [hunk 2]
```diff
@@ -13,7 +13,7 @@ public override void OnInspectorGUI()
         serializedObject.Update();
 
         EditorGUILayout.PropertyField(serializedObject.FindProperty("TransmissionTargetToken"));
-        EditorGUILayout.PropertyField(serializedObject.FindProperty("AppCenterStartupType"));
+        //EditorGUILayout.PropertyField(serializedObject.FindProperty("AppCenterStartupType"));
```
Comment (max-mironov) [inline Assets/AppCenter/Editor/AppCenterSettingsEditorAdvanced.cs:17]: @annakocheshkova Can you please add a comment why we need to keep this commented and do not delete this at all?

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs (hunk 1 above)
Comment (annakocheshkova) [reply Assets/AppCenter/Editor/AppCenterPreBuild.cs:130]: The methods use `settingsMaker` of different types

Comment (max-mironov) [reply Assets/AppCenter/Editor/AppCenterPreBuild.cs:130]: gotcha!

**Review by max-mironov — APPROVED** [review]

**Pull request merged**